lib/wasix/src/os/console/mod.rs

```diff
@@ -298,3 +280,65 @@ impl Console {
             .ok();
     }
 }
+
+#[cfg(test, not(target_family = "wasm"))]
```

Comment:
Should we be using `all()` here?

```suggestion
#[cfg(all(test, not(target_family = "wasm")))]
```